packages/shared/components/TextHint.svelte

```diff
@@ -0,0 +1,16 @@
+<script lang="typescript">
+    import { Icon, Text } from 'shared/components'
+
+    export let classes: string = ''
+    export let icon: 'info' | 'exclamation' | 'none' = 'none'
+    export let hint: string = ''
+</script>
+
+{#if hint !== ''}
```

Comment:
empty strings are falsey so could just write:
```suggestion
{#if hint}
```